lib/pbench/server/__init__.py

```diff
@@ -143,6 +115,55 @@ def mocked_time():
             + " ".join([f"WONT-INDEX.{i:d}" for i in range(1, 12)])
         )
 
+    @property
+    def server_config(self):
+        return self.conf["pbench-server"]
+
+    @property
+    def es_url(self):
+        try:
+            return f"http://{self.conf.get('elasticsearch', 'host')}:{self.conf.get('elasticsearch', 'port')}"
+        except (NoOptionError, NoSectionError):
+            return ""
+
+    @property
+    def index_prefix(self):
+        try:
+            return self.conf.get("Indexing", "index_prefix")
+        except (NoOptionError, NoSectionError):
+            return ""
+
```

Comment:
I believe we might need these properties as we add more es queries

Reply:
The question is would we have code written in another module that talks to Elasticsearch NOT use the `elasticsearch_api` module?

The query_controllers module also talks to Elasticsearch, but why not make one module which handles the communication for both?